### ISSUE TITLE
Investigate an AutoSwitch for users who declare they are using screen readers

### ISSUE THREAD

**Issue by PatReynoldsFUG** [FreeUKGen/FreeBMD2#532]
For features we cannot make available to screen readers, we should have a way of switching them off (e.g. pop-up dropdown lists) and (where possible) switching on a screen-reader friendly alternative

**Comment by DeniseColbert** [comment]
Any text should be in plain sight for screen readers to have it read out to them
